CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [18.0.0]
+### Added
+- **BREAKING**: ERC1155 support ([#615](https://github.com/MetaMask/controllers/pull/615))
+  -  `CollectiblesController` requires `getOwnerOf`, `balanceOfERC1155Collectible` and `uriERC1155Collectible` properties in the constructor which are methods from `AssetsContractController`.
+- Custom network + IPFS support ([#616](https://github.com/MetaMask/controllers/pull/616))
+
+### Changed
+- Remove unnecessary return statements ([#606](https://github.com/MetaMask/controllers/pull/606))
+- Bump @metamask/contract-metadata from 1.29.0 to 1.30.0 ([#607](https://github.com/MetaMask/controllers/pull/607))
+- Remove redundant ESLint rule ([#604](https://github.com/MetaMask/controllers/pull/604))
+- Fix return type of `fetchGasFeeEstimates` ([#602](https://github.com/MetaMask/controllers/pull/602))
```

Comment:
honestly I think all of these `Changed` logs fall under `Generally any changes that don't affect consumers of the package (e.g. lockfile changes or development environment changes) are omitted`.

Reply:
I would keep the `Bump` log, they seen to be standard, I agree to remove the other ones